web/source/keymanweb.js

```diff
@@ -3681,21 +3722,30 @@
     } else {
       if(Pelem.tagName.toLowerCase() != 'iframe') {
         if(keymanweb.isKMWInput(Pelem)) {
-          var x = keymanweb.setupTouchElement(Pelem);
-
-          if(x) {
-            // if needed, let the timeout variable be: keymanweb.touchInputAddedTimer
-            window.setTimeout(function() {
-              keymanweb.updateInput(x);
-            }, 1);
-          }
+          keymanweb.setupTouchElement(Pelem);
```

Comment:
Also, are we sure we want to move toward a _more_ monolithic function?  I feel the design presented here helps with code clarity.